src/utils.jl

```diff
@@ -0,0 +1,21 @@
+# A couple utility methods to avoid messing up var and obs dimensions
+obsdim(dims) = dims
```

Comment:
this needs a comment.
I don't understand why that is correct

Reply:
I think it was correct for our use cases and was copied from https://github.com/invenia/Impute.jl/blob/master/src/imputors.jl#L11 NOTE: This is copied to `deprecated.jl` in a later commit when it made sense to deprecate `DropObs` and `DropVars` in favour of a simple `filter` w/ `dims`.